src/app/shared/components/cities-search/cities-search.component.ts

```diff
@@ -30,7 +30,7 @@ export class CitiesSearchComponent implements OnInit, ControlValueAccessor {
   @Input() label = '';
   @Input() required: boolean = false;
   @Input() isRegionPicked: boolean = false;
-
+  @ViewChild('autoCompleteInput') locationInput!: ElementRef<HTMLInputElement>;
```

Comment:
te dwa `ViewChild`y mają taki sam string przekazany, nie da się tego napisać razem, albo czy to nie jest jakiś problem?

Reply:
To dziwnie wyglada ale działa, a linter nie krzyczy, chciałem dać możliwość dwóch typów do jednego ale nie działa:
<img width="902" alt="image" src="https://user-images.githubusercontent.com/12514322/164735522-80a190c5-2869-4646-b07c-6940510fc499.png">
<img width="848" alt="image" src="https://user-images.githubusercontent.com/12514322/164735575-e5355858-87cf-45ce-a073-ea180d199ca8.png">
Więc zostaje tak, albo mogę dać <any> ale nie wiem czy to jest lepsze wyjście.

Reply:
Ja się na angularze aż tak nie znam :) jeśli działa to może być